packages/dds/tree/src/simple-tree/toMapTree.ts

```diff
@@ -726,6 +726,7 @@ function tryGetInnerNode(target: unknown): InnerNode | undefined {
  * Content which can be used to build a node.
  * @remarks
  * Can contain unhydrated nodes, but can not be an unhydrated node at the root.
+ * @system @alpha
  */
 export type FactoryContent =
```

Comment:
Since these are being used elsewhere now, I think moving them to `schemaTypes.ts` is warranted (fine to do in a different PR though if you want to avoid churn.)

Reply:
simple-tree/api (where schema types are) depends on simple-tree/*, and the reverse is not allowed, so moving these would make it invalid to use them here for their current usage.

Reply:
Yea that's fine. They're not in a bad place now, just a little opportunity for future cleanup/organizing.